configs/components/leatherman.json

```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/leatherman.git", "ref": "0.8.1"}
```

Comment:
ping @MikaelSmith on this change, do we need leatherman 0.9.0 for ruby 2.3? If not I would like to keep leatherman 0.8.1 for the next release.